### ISSUE TITLE
Accessibility fix: Conservatory Videos

### ISSUE THREAD

**Issue by senaida1097** [senaida1097/CISC4900Project#6]
- The sound file graphics on this page are not accessible becuase of    insufficient color contrast.
- Fix the issue by replacing the links to those files with buttons.
